packages/php-wasm/logger/src/lib/logger.ts

```diff
@@ -76,7 +76,12 @@ export class Logger extends EventTarget {
 	 * @param PromiseRejectionEvent event
 	 */
 	private logUnhandledRejection(event: PromiseRejectionEvent) {
-		this.log(`${event.reason.stack}`, 'Error');
+		// No reason was provided, so we can't log anything.
```

Comment:
Should this confirm the reason is not PHP fatal error but WASM crash? Or no need?

Reply:
Yes, that's a great point! I will take some extra time to test it with the [memory leak example](https://github.com/WordPress/wordpress-playground/issues/1128) and see if we can get some insights from the error.

Reply:
I couldn't get any extra info here, so I will merge this PR as is.

Reply:
This is technically a request error and the code is working as expected. I'm searching now what PHP exit code 255 exactly means and if this is something we can use as a differentiator between regular PHP errors and PHP-WASM errors.